bundle/PostInstall.cmake

```diff
@@ -0,0 +1,25 @@
+set(examples "copy_stencil;hori_diff;tridiagonal_solve")
```

Comment:
when I run 
```
cmake ..
make install
```
this breaks. If using 
```
cmake .. -DDAWN_PYTHON=ON
make install
```
it works. we should probably check if it was set to on and do nothing otherwise?